node/sc/bridge_manager.go

```diff
@@ -156,6 +156,19 @@ func NewBridgeInfo(sb *SubBridge, addr common.Address, bridge *bridgecontract.Br
 	return bi, nil
 }
 
+// handleValueTransferLog pushes value transfer transaction's log
+func handleValueTransferLog(isChild bool, funcName, txHash string, reqNonce uint64) {
```

Comment:
```suggestion
func handleValueTransferLog(toChild bool, funcName, txHash string, reqNonce uint64) {
```

What about changing 'isChild' to 'toChild'?

Reply:
In the implementation of this function shows that this variable actually indicates the direction. What is the reason that you think that 'is' is more appropriate?

Reply:
```suggestion
func handleValueTransferLog(onChild bool, funcName, txHash string, reqNonce uint64) {
```

What about this? It seems that call site uses the name of 'onChildChain'.

Reply:
Applied.